test/unit.test.js

```diff
@@ -201,7 +223,12 @@ for (const { testName, isRoot } of unitTests) {
     expect(nftCache.statCache).toBeDefined()
     expect(nftCache.symlinkCache).toBeDefined()
     expect(nftCache.analysisCache).toBeDefined()
-    await doTrace(true)
+    
+    try {
+      await doTrace(true)
+    } finally {
+      console.error(`Failed for cached run`)
```

Comment:
@ijjk Should this be `catch` instead of `finally`?

Reply:
Should be finally so it still throws, the log here was meant for additional context when it fails.

Reply:
But finally runs every time, not just failures like catch does.

Look at CI and it’s printed every time https://github.com/vercel/nft/runs/7420198326?check_suite_focus=true#step:9:120

maybe I misunderstood

Reply:
Oh yeah 🤦 can move to catch an manually rethrow

Reply:
updated in https://github.com/vercel/nft/pull/304/commits/51ea58a48dab5e4d2742815ca483448c9dbcba82